packages/backend/discovery/allbridge/ethereum/config.jsonc

```diff
@@ -11,7 +11,11 @@
     "0x203e8785b4d4312c4152D0c42Ba3FA8BD79086dA", // allbridgeMessenger
     "0x7f02294f065A605C8D21D9812b8d1De7E5E1da74", // wormholeMessenger
     "0xC51397b75B783E31469bFaADE79913F3f82210d6", // cctp adapter
-    "0x0BdF6139F2841A7856Ca154D851182C52F5b96e0" // gasoracle
+    "0x0BdF6139F2841A7856Ca154D851182C52F5b96e0", // gasoracle
+
+    "0x7234dB900E907398EdfAdA744d5Bf8A842B335BA",
+    "0x26f9AA5a00825d37E4ebBa0844fcCF1f852640D5",
+    "0xb7C522Adb3429e2C7474df324c7a3744A5803414"
```

Comment:
// This was hardcoded in the .ts file as CCTP manager. Putting this here until it can be automatically discovered.